x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_list_filters.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { memo, useCallback, useMemo } from 'react';
```

Comment:
You see how things get confusing?
You have a `action_list_filters` (plural) that contains the entire set of filters and it includes a `action_list_filter` (singular).

Reply:
renamed 22e0c598c6bdfbb92de18cd6cc7d427c8dae32f9